meta-lmp-support/README.md

```diff
@@ -0,0 +1,5 @@
+# Meta for Linux microplatform (LmP)
+
+This meta-layer contains required support recipes to build and run the Pelion Edge on LmP, a Linux based platform from [Foundries.io](https://foundries.io/).
+
+For more information, please see the [Pelion Edge documentation](https://www.pelion.com/docs/device-management-edge/latest/introduction/index.html), and [the quickstart for LmP](https://www.pelion.com/docs/device-management-edge/latest/quick-start/lmp-quick-start.html)
```

Comment:
@AnotherButler Does this kind of documentation need to be on your radar?

Reply:
This documentation will be added in dm-docs PR 102